grails-app/views/consentGroupDocuments/_index.gsp

```diff
@@ -0,0 +1,24 @@
+<g:javascript>
+    const user = {
+        <g:if test="${session?.user}">
+            displayName: "${session.user.displayName}",
+            email: "${session.user.emailAddress}",
+            userName: "${session.user.userName}"
+        </g:if>
+    };
+    const component = {
+        user: user,
+        issue: '${issue.requestDate}',
+        projectKey:'${issue.projectKey}',
```

Comment:
Also see that same comment. I'd like to try and abstract out anything that is a variable to a service call to get the variable value at the time the component needs to load it. 

Also, `issue: '${issue.requestDate}',` is confusing. Is it `issue` or is it `requestDate` that you want here?